docs-src/content/getting-started/recognize.md

```diff
@@ -15,6 +15,7 @@ recognition.
 <!--more-->
```

Comment:
This comment is unrelated to your PR but If streaming recognition is the preferred method to use cubic, I'd put that page first above this one (I'm not sure how the ordering of the pages is configured). Then I'd have a preamble here which says that 'If your audio is short you may prefer this way...'.